docarray/array/mixins/io/pushpull.py

```diff
@@ -250,6 +250,8 @@ def _get_chunk(_batch):
         if response.ok:
             return response.json()['data']
         else:
+            if response.status_code == 403:
```

Comment:
403 seems a magic number, lets use a constant

Reply:
@jina-ai/team-hubble @delgermurun @mapleeit these constants should come from hubble-sdk

Reply:
Does it make sense to use `http.HTTPStatus.FORBIDDEN` here? 

It's standard http status code and `libarary/http` already provides that.

Reply:
The `403` error is not the only one. How about trying to get a message from every error response?
All errors should have a message, but checking for its existence is also helpful to have.

Reply:
applied

Reply:
Agree with @delgermurun 

For all the status code `>= 400`, we should adopt the same strategy, which is printing the error message out.

Reply:
done